src/GraphQLParser.ApiTests/GraphQLParser.approved.txt

```diff
@@ -469,8 +499,12 @@ namespace GraphQLParser.AST
     public class GraphQLStringValue : GraphQLParser.AST.GraphQLValue
     {
         public GraphQLStringValue() { }
+        public GraphQLStringValue(string value) { }
+        public override object? ClrValue { get; }
         public override GraphQLParser.AST.ASTNodeKind Kind { get; }
+        public string TypedValue { get; }
```

Comment:
Even here `TypedValue` is exposed, but not `IntValue`, `LongValue`, etc.

Reply:
reverted